api/server/server.go

```diff
@@ -351,10 +351,24 @@ func getContainersChanges(eng *engine.Engine, version version.Version, w http.Re
 	if vars == nil {
 		return fmt.Errorf("Missing parameter")
 	}
-	var job = eng.Job("container_changes", vars["name"])
-	streamJSON(job, w, false)
 
-	return job.Run()
+	d := getDaemon(eng)
+	cont, err := d.Get(vars["name"])
```

Comment:
I think we may want to check to see if "name" is empty to return a nicer error message.